doc/releases/changelog-dev.md

```diff
@@ -4,192 +4,331 @@
 
 <h3>New features since last release</h3>
 
-* Support drawing QJIT QNode from Catalyst.
-  [(#4609)](https://github.com/PennyLaneAI/pennylane/pull/4609)
+<h4>Decompose circuits into the Clifford+T gateset 🧩</h4>
 
-  ```python
-  import catalyst
+* A new transform called `clifford_t_decomposition` is available, which decomposes
+  circuits into the Clifford+T gate set. 
+  [(#ABCD)]()
```

Comment:
Just putting a note here that the PR numbers need to be added once they're merged